src/pages/New.js

```diff
@@ -28,12 +28,13 @@ export default function New() {
       title: title,
       categories: cuisines,
       distance: 6,
-      rating: 3.1,
+      rating: qualityRating,
       description: "Eat italian"
     });
   }
 
   const [title, setTitle] = useState("");
+  const [qualityRating, setQualityRating] = useState("0");
```

Comment:
If `qualityRating` is a number, please set the default state to a number too (`useState(0)`).
Afterward, you don't need to convert the `qualityRating` to a Number in L: 78.

Reply:
Should also be fixed now